treeherder/webapp/api/performance_serializers.py

```diff
@@ -155,6 +158,7 @@ class PerformanceAlertSummarySerializer(serializers.ModelSerializer):
 
     def update(self, instance, validated_data):
         instance.timestamp_first_triage()
+        logger.warning(f"validated_data: {validated_data}")
```

Comment:
I don't follow why you'd log a warning whenever this is updated. Can you add a clearer message or add a comment?